### ISSUE TITLE
[Feature] Options to force april fools modes: ehT dlekS, horse mode, long neck mode

### ISSUE THREAD

**Issue by heavyutils** [g0aty/SickoMenu#47]

**Comment by g0aty** [comment]
I might as well add this since it's client sided

**Comment by heavyutils** [comment]
> I might as well add this since it's client sided

ehT dlekS isn't client sided (afaik) but sure buddy

**Comment by g0aty** [comment]
I was meaning that the bodies are client sided but dleks is server sided.

**Comment by heavyutils** [comment]
Yup, horse mode fully is. And yet again, long neck isn't fully client side. But ofc everything is better when it can be serverside! Because everyone took effect of the long neck mode when you enabled or disabled it in that time

**Comment by g0aty** [comment]
It won't really be possible to do body types server-side as long mode depends on the player's device time.
